Doom_CooldownPulse.lua

```diff
@@ -374,6 +376,7 @@ function DCP:CreateOptionsFrame()
         { text = "Icon Size", value = "iconSize", min = 30, max = 125, step = 5 },
         { text = "Fade In Time", value = "fadeInTime", min = 0, max = 1.5, step = 0.1 },
         { text = "Fade Out Time", value = "fadeOutTime", min = 0, max = 1.5, step = 0.1 },
+        { text = "Show Before Available (Seconds)", value = "remainingCooldownWhenNotified", min = 0, max = 3, step = 1 },
```

Comment:
I might suggest smaller step increments here in case someone wants to be precise, like `0.1`.

Also, maybe we use "Time" in the label for consistency with the other options? Unsure if this might also help avoid increasing the width below.

```suggestion
        { text = "Show Before Available Time", value = "remainingCooldownWhenNotified", min = 0, max = 3, step = 0.1 },
```